bin/coreos-install

```diff
@@ -22,10 +34,7 @@ fi
 : ${VERSION_ID:=current}
 CHANNEL_ID=${GROUP:-stable}
 
-if [[ -e /usr/share/coreos/release ]]; then
-    BOARD=$(gawk --field-separator '=' '/COREOS_RELEASE_BOARD=/ { print $2 }' /usr/share/coreos/release)
-fi
-BOARD=${BOARD:-"amd64-usr"}
+BOARD=${BOARD:-$(default_board)}
```

Comment:
Since the `BOARD` variable is no longer defined above, this line could just be `BOARD=$(default_board)`.  I think I'd prefer this to not allow an environment variable to affect the value.